resources/crds/ripsaw_v1alpha1_couchbase_cr.yaml

```diff
@@ -1,42 +1,42 @@
-apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
-kind: Benchmark
-metadata:
-  name: couchbase-infra
-  namespace: ripsaw
-spec:
-  infrastructure:
-    name: couchbase
-    args:
-      servers:
-        # Typical deployment size is 3
-        size: 3
-      storage:
-        use_persistent_storage: True
-        class_name: "rook-ceph-block"
-        volume_size: 10Gi
-      deployment:
-        # These are deployment defaults from roles/couchbase-infra/defaults
-        # that can be overridden here.
-        #
-        ## For Generic K8s w/ Upstream OLM and Marketplace Defaults
-        #catalogsource_namespace: "marketplace"
-        #cb_operator_package: "couchbase-enterprise"
-        #pod_base_image: "docker.io/library/couchbase"
-        #pod_version: "enterprise-5.5.2"
-        #default_bucket_password: "password"
-        #
-        ## For OpenShift v3 w/ Upstream OLM and Marketplace Defaults
-        #catalogsource_namespace: "marketplace"
-        #cb_operator_package: "couchbase-enterprise"
-        #pod_base_image: "registry.connect.redhat.com/couchbase/server"
-        #pod_version: "5.5.4-1"
-        #default_bucket_password: "password"
-        #pull_secret_name: "rhsecret"
-        #
-        ## For OpenShift v4 w/ Built-In OLM and Marketplace
-        #catalogsource_namespace: "openshift-marketplace"
-        #cb_operator_package: "couchbase-enterprise-certified"
-        #pod_base_image: "registry.connect.redhat.com/couchbase/server"
-        #pod_version: "5.5.4-1"
-        #default_bucket_password: "password"
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1	
```

Comment:
I honestly don't know wh this even happened. Like there's no change. I even tried to redo copy-paste but this still shows as a change. I tried with and without editor too.

Reply:
there is an "space" at end, I found that having space in .yaml files can be sometimes problematic and I usually open files in `vim` and then run `:%s/\s\+$//e`  this will remove all trailing spaces in file, also check https://github.com/kubernetes/kubernetes/issues/36222